company_website/templates/common_partials/head.haml

```diff
@@ -1,5 +1,14 @@
 {% load static %}
 
+/ Google Tag Manager
+:javascript
+  (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','GTM-TCWZK7N');
+/ End Google Tag Manager
+
```

Comment:
This should be in different `.js` file and then it should be imported in `head.haml`